src/mgr/MgrOpRequest.cc

```diff
@@ -28,7 +28,7 @@ using ceph::Formatter;
 
 MgrOpRequest::MgrOpRequest(MessageRef req, OpTracker* tracker)
     : TrackedOp(tracker, req->get_recv_stamp()),
-      request(req) {
+      request(req->get()) {
```

Comment:
This doens't look right. `MgrOpRequest::request` is a smart pointer?

Reply:
@batrick The dtor had an explicit dropping of ref of the request object.
I've done away with that statement and things look fine in QA runs.
We need more widespread testing for this though.